CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v0.31.5
+
+*April 16th, 2019*
+
+This release fixes a regression from v0.31.4 where, in existing chains that
+were upgraded, `/validators` could return an empty validator set. This is true
```

Comment:
Suggestion: 
"This release fixes a regression from v0.31.4 where, in existing chains that
were upgraded, querying `/validators` via RPC could return an empty validator set. 
This happened for almost all height, given the validator set remained the same."
?

Reply:
I like my version better :) I'll let @ebuchman be the judge

Reply:
Ismail's is defntly clearer that this is about the RPC endpoint, though we often refer to `/xxx` and expect the reader to know we're talking about the RPC.